### PR TITLE
Restraint max search results to 2 for newbies (younger than 2 weeks)

### PR DESCRIPTION
**Pull request by joonhocho**
@2is10 @stkem 
Tested and it, at least, does not break anything.
I don't know how to test e2e with new backend.

Feel free to modify or merge it.

### CODE REVIEW

File: packrat/main.js [hunk 1]
```diff
@@ -1167,6 +1167,28 @@ function tellTabsUnreadThreadCountIfChanged(td) { // (td, url[, url]...)
   forEachTabAt.apply(null, args);
 }
 
+
+var MAX_RES = 5;
```
Comment (2is10) [inline packrat/main.js]: Might want to rename `MAX_RES` → `DEFAULT_RES` since the max is actually 9.

File: packrat/main.js [hunk 2]
```diff
@@ -1167,6 +1167,28 @@ function tellTabsUnreadThreadCountIfChanged(td) { // (td, url[, url]...)
...
+    return MAX_RES;
+  }
+
+  var pref = api.prefs.get("maxResults");
+  if (pref !== MAX_RES) {
+    return pref;
+  }
+
+  var joined = session.joined;
+  if (joined && (new Date() - joined) < TWO_WEEKS) {
```
Comment (2is10) [inline packrat/main.js]: `new Date()` → `Date.now()` avoids an unnecessary object creation

**Comment by 2is10** [comment]
Looks good to me. I just merged the shoebox part. I’ll deploy it once the build passes.

**Comment by 2is10** [comment]
Not sure what happened to this pull request. It’s showing 73 commits now. Might want to try pulling from FortyTwoEng master and pushing to your master again to clear it up.

File: packrat/main.js (hunk 1 above)
Comment (joonhocho) [inline packrat/main.js]: fixed

File: packrat/main.js (hunk 2 above)
Comment (joonhocho) [inline packrat/main.js]: fixed

**Comment by joonhocho** [comment]
It seems fine to me.
[image: Inline image 1]

On Wed, Dec 4, 2013 at 9:27 PM, Jared Jacobs notifications@github.comwrote:

> Not sure what happened to this pull request. It’s showing 73 commits now.
> Might want to try pulling from FortyTwoEng master and pushing to your
> master again to clear it up.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/FortyTwoEng/keepit/pull/5118#issuecomment-29872583
> .

## 

Regards,
Joon Ho Cho

**Comment by joonhocho** [comment]
It seems fine to me, but I will wait til tomorrow just to be sure.

**Pull request merged**